### PR TITLE
adapt cli command get to use Illumina models

### PR DESCRIPTION
**Pull request by diitaz93**
## Description

Refactors the `cg get flow-cell` command into `cg get sequencing-run`.
### Changed

- Replaced the old filter functions with the new ones
- replaced all ocurrences
- fixed tests

### How to prepare for test

- [x] Ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] Paxa the environment: `paxa`
- [x] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b adapt-cli-get-illumina -a
    ```

### How to test

- [x] Do `cg get --help`
```shell
$ cg get --help
Usage: cg get [OPTIONS] COMMAND [ARGS]...

  Get information about records in the database.

Options:
  -i, --identifier TEXT  made a guess what type you are looking for
  --help                 Show this message and exit.

Commands:
  analysis        Get information about case analysis.
  case            Get information about a case.
  relations       Get information about case relations.
  sample          Get information about a sample.
  sequencing-run  Get information about a sequencing run and its samples, if any.
[js.diazboada@hasta:~] [S_base] $ 
```

### Expected test outcome

- [x] Check that the command works as expected

## Review

- [x] Tests executed by SD
- [x] "Merge and deploy" approved by CO, VJ
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

Merging to dev branch, not going to deploy yet

### CODE REVIEW

File: cg/cli/get.py [hunk 1]
```diff
@@ -6,15 +6,22 @@
...
 from cg.models.cg_config import CGConfig
-from cg.store.models import Case, Customer, Flowcell, Sample
+from cg.store.models import Case, Customer, IlluminaSequencingRun, Sample
 from cg.store.store import Store
 
 LOG = logging.getLogger(__name__)
 ANALYSIS_HEADERS = ["Analysis Date", "Workflow", "Version"]
 FAMILY_HEADERS = ["Case", "Name", "Customer", "Priority", "Panels", "Action"]
-FLOW_CELL_HEADERS = ["Flowcell", "Type", "Sequencer", "Date", "Archived?", "Status"]
+FLOW_CELL_HEADERS = [
```
Comment (ChrOertlin) [inline cg/cli/get.py:16]:
```suggestion
SEQUENCING_RUN_HEADERS = [
```

File: cg/cli/get.py [hunk 2]
```diff
@@ -6,15 +6,22 @@
...
+    "Flowcell",
+    "Type",
+    "Sequencer",
+    "Sequencing started",
+    "Sequencing finished",
+    "Archived?",
+    "Status",
```
Comment (ChrOertlin) [inline cg/cli/get.py:23]:
```suggestion
    "Data availability",
```

File: cg/cli/get.py [hunk 3]
```diff
@@ -6,15 +6,22 @@
...
 ANALYSIS_HEADERS = ["Analysis Date", "Workflow", "Version"]
 FAMILY_HEADERS = ["Case", "Name", "Customer", "Priority", "Panels", "Action"]
-FLOW_CELL_HEADERS = ["Flowcell", "Type", "Sequencer", "Date", "Archived?", "Status"]
+FLOW_CELL_HEADERS = [
+    "Flowcell",
+    "Type",
+    "Sequencer",
+    "Sequencing started",
+    "Sequencing finished",
+    "Archived?",
```
Comment (ChrOertlin) [inline cg/cli/get.py:22]:
```suggestion
    "Archived?", 
```
Do we use this?

File: tests/cli/get/test_cli_get_flow_cell.py [hunk 4]
```diff
@@ -7,159 +7,203 @@
...
-    result = cli_runner.invoke(get, ["flow-cell", "does_not_exist"], obj=base_context)
+    # WHEN getting a non-existent sequencing run
+    result = cli_runner.invoke(get, ["sequencing-run", "does_not_exist"], obj=base_context)
 
-    # THEN it should complain about missing flow cell instead of getting a flow cell
+    # THEN the command exits with a non-zero exit code
     assert result.exit_code == EXIT_FAIL
 
 
 def test_get_flow_cell_required(
```
Comment (ChrOertlin) [inline tests/cli/get/test_cli_get_flow_cell.py:26]:
```suggestion
def test_get_sequencing_run_required(
```

**Review by ChrOertlin — APPROVED** [review]
Looks good!

File: cg/cli/get.py (hunk 3 above)
Comment (diitaz93) [reply cg/cli/get.py:22]: KSV suggests keeping it just in case

**Review by Vince-janv — APPROVED** [review]
Clean ✨

**Pull request merged**